<html>
<body>
<center>
<h2>POSTERS</h2>

<b>Note to presenters:</b> 
<br>
Odd-numbered posters go Friday. 
Even-numbered posters go Saturday.
<br>
Poster boards are 8 feet wide and 4 feet high.
Bring push-pins.
<p>
<table width=80% border=2>
<tr>
<td><b>Assignment</b>
<td><b>Title</b>
<td><b>Authors</b>
</tr>
</tr>
</html>
<tr>
<td>SF-39
<td>Extracting Simple Motifs Under the Levenshtein Measure
<td>Adebiyi, Ezekiel F.
</tr>
</tr>

<tr>
<td>SA-1
<td><i>MaM</i>: Multiple Alignment Manipulator
<td>Alkan, Can and T&uuml;z&uuml;n, Eray and Eichler, Evan and Bailey, Jeff and Sahinalp, Cenk S.
</tr>
</tr>

<tr>
<td>IN-125
<td>NIGMS Funding Opportunities for Complex Biomedical Systems Research and Training
<td>Anderson, James J.
</tr>

<tr>
<td>SA-2
<td>G-language Genome Analysis Environment
<td>Arakawa, Kazuharu and Mori, Koya and Tomita, Masaru
</tr>

<tr>
<td>RR-85
<td>Protein Families and Regulatory Patterns amongst the Transcription Factors of <i>E. coli</i>
<td>Babu, M. Madan and Teichmann, Sarah A.
</tr>

<tr>
<td>SA-3
<td><i>mreps</i>: A Program for Exhaustive Search for Tandem Repeats in DNA Sequences
<td>Bana, Ghizlane and Kolpakov, Roman and Giraud, Mathieu and Rabbat, Ralph and Kucherov, Gregory
</tr>

<tr>
<td>PW-86
<td>Combinatorial Analysis of Transcription Regulation in Yeast Cell-cycle
<td>Banerjee, Nila and Zhang, Michael Q.
</tr>

<tr>
<td>ME-47
<td>An EST Cluster-Sorting Software HSAnalyst as an Universal Tool for Discovering of Genes with Unusual Expression
<td>Baranova, A.V. and Lobashev, A.V. and Ivanov, D.V. and Krukovskaya, L.L. and Zinchenko, V.V. and Shestakov, S.V. and Yankovsky, N.K. and Kozlov, A.P.
</tr>

<tr>
<td>SA-4
<td>Building a Genome Database Using an Object-Oriented Approach
<td>Barbasiewicz, Anna and Liu, Lin and Lang, B.Franz and Burger, Gertruad
</tr>

<tr>
<td>ME-48
<td>How to Arrange Gene Expression Profiles in Linear Orders
<td>Bei&szlig;barth, Tim and Vingron, Martin
</tr>

<tr>
<td>RR-74
<td>The <i>E. coli</i> Promoter Region: A Statistical Analysis
<td>Bejerano, Gill and Margalit, Hanah and Tishby, Naftali
</tr>

<tr>
<td>PA-112
<td>Enumerating Optimal Sequences of Reversals: Tools and Experiments
<td>Bergeron, Anne and Chauve, Cedric and Hartman, Tzvika and Saint-Onge, Karine
</tr>

<tr>
<td>PS-94
<td>Ligand Binding to the Pregnane X Receptor by Geometric Matching of Hydrogen Bonds
<td>Berretty, Robert-Paul and Hsu, David and Kettner, Lutz and Mascarenhas, Ajith and Redinbo, Matthew R. and Snoeyink, Jack and Watkins, Ryan
</tr>

<tr>
<td>PA-113
<td>MGR - A Web Tool for Multiple Genome Rearrangements
<td>Bourque, Guillaume and Tesler, Glenn
</tr>

<tr>
<td>SA-5
<td>A Supervised Learning Approach for Detecting Significant Local Alignments
<td>Breimer, Eric A. and Goldberg, Mark K.
</tr>

<tr>
<td>SA-6
<td>CHAOS: A Heuristic Algorithm for Local Alignment
<td>Brudno, Michael and Morgenstern, Burkhard
</tr>

<tr>
<td>PA-114
<td>Mitochondrial Eve and Population Bottlenecks
<td>Buss, Samuel R. and Clote, Peter G.
</tr>

<tr>
<td>SA-7
<td>A Fast String Matching Method for Database Queries
<td>Campbell, Shannon R.
</tr>

<tr>
<td>PA-115
<td>Genomic Signature: A Global Sequence Analysis Concept Applied to Phylogeny
<td>Chapus, Charles and Fertil, Bernard and Giron, Alain and Deschavanne, Patrick
</tr>

<tr>
<td>RR-75
<td>Characterization and Prediction of the Nanos Response Element with Sequence Analysis in Drosophila
<td>Chen, Gengxin and Zhang, Michael Q.
</tr>

<tr>
<td>SN-44
<td>SNP Mapping Using Genome-wide Marker Sequences
<td>Chen, Leslie Y.Y. and Lu, Szu-Hsien and Shih, Edward S.C. and Hwang, Ming-Jing
</tr>

<tr>
<td>RR-76
<td>Finding Recurrent Binding Site Clusters
<td>Cheng, Yizong and Aronow, Bruce
</tr>

<tr>
<td>RR-77
<td>PromoterFootprint: A New Method for Alignment of Regulatory Genomic Sequences. Phylogenetic Footprinting of TF Binding Sites.
<td>Cheremushkin, Eugene and Kel, Alexander
</tr>

<tr>
<td>RR-78
<td>Quantifying Sequence Correlates of Gene Regulation
<td>Clarke, Neil D. and Granek, Joshua A. and Liu, Xiao
</tr>

<tr>
<td>SA-8
<td>Pooled Genomic Indexing
<td>Cs&uuml;r&ouml;s, Mikl&oacute;s and Milosavljevic, Aleksandar
</tr>

<tr>
<td>PW-87
<td>Predicting Protein Interaction Paths and Multimeric Complex Configuration: A Binding Site Recognition Based Approach
<td>Del Carpio-Mu&ntilde;oz, Carlos A.
</tr>

<tr>
<td>SA-9
<td>Discovering Common Mutually-Correlated Pairs of Positions for the Families of Immunoglobulin Fold
<td>Dement'ev, Andrey and Galitsky, Boris
</tr>

<tr>
<td>SW-123
<td>A User Friendly Bioinformatical Pipeline Environment
<td>Dunbar, Donald and Proutski, Vitali
</tr>

<tr>
<td>ME-49
<td>A Variance-Stabilizing Transformation for Microarray Data
<td>Durbin, Blythe and Hardin, Johanna and Hawkins, Douglas and Rocke, David
</tr>

<tr>
<td>LA-45
<td>Superlink: A New Program for Exact Genetic Linkage Analysis of General Pedigrees
<td>Fishelson, Ma'ayan and Geiger, Dan
</tr>

<tr>
<td>SA-10
<td>Substitution Matrices Derived from Structurally Constrained Protein Evolution Simulations
<td>Fornasari, Maria and Parisi, Gustavo and Echave, Juli&aacute;n
</tr>

<tr>
<td>PW-88
<td>Inferring Gene Networks from Perturbed Expression Profiles
<td>de la Fuente, Alberto and Brazhnik, Paul and Mendes, Pedro
</tr>

<tr>
<td>SA-11
<td>Contextual Alignment of Biological Sequences
<td>Gambin, Anna and Lasota, Slawomir and Szklarczyk, Radoslaw and Tiuryn, Jerzy and Tyszkiewicz, Jerzy
</tr>

<tr>
<td>SA-12
<td>Substitution Tables for Contextual Alignment
<td>Gambin, Anna and Tyszkiewicz, Jerzy
</tr>

<tr>
<td>SA-13
<td>Rare and Frequent n-Grams in Whole-Genome Protein Sequences
<td>Ganapathiraju, Madhavi and Klein-Seetharaman, Judith and Rosenfeld, Roni and Carbonell, Jaime and Reddy, Raj
</tr>

<tr>
<td>SA-14
<td>`DNA Kinases' -- Computational Insights
<td>Ganji, Gopinath and Li, Yingfu and Cuticchia, A. Jamie
</tr>

<tr>
<td>RR-79
<td>Promoter Prediction Using Hidden Markov Model
<td>Gao, Feng and Goldstein, Richard A.
</tr>

<tr>
<td>ME-50
<td>Measurement Error and Reproducibility of Affymetrix Microarray Data with Application to Autism
<td>Geller, Sue and Zhou, Lei and Gregg, Jeff and Hagerman, Paul and Rocke, David M.
</tr>

<tr>
<td>PS-95
<td>A Fast Pattern-Matching System on a Protein Topology Database
<td>Gilbert, David and Torrance, Gilleain and Viksna, Juris and Dhulia, Neha and Mamtora, Darshit and Westhead, David and Michalopoulos, Ioannis
</tr>

<tr>
<td>SA-15
<td>Improving Shotgun Sequencing Through Learning
<td>Goldberg, Mark and Lim, Darren
</tr>

<tr>
<td>SA-16
<td>Closest Strings, Primer Design, and Motif Search
<td>Gramm, Jens and H&uuml;ffner, Falk and Niedermeier, Rolf
</tr>

<tr>
<td>ME-51
<td>Performance Analysis of Differential Expression Prediction Algorithms Using Simulated Array Data
<td>Grant, Gregory and Manduchi, Elisabetta and Sokolovski, Sam and Stoeckert, Chris
</tr>

<tr>
<td>ME-52
<td>Discovery of Correlation of Tissue Gene Expression Using cDNA Libraries
<td>Horng, Jorng-Tzong and Huang, Hsien Da and Tseng, Kuo-Yen and Tsou, Tsung-Shan and Liu, Baw-Jhiune and Kao, Cheng-Yan
</tr>

<tr>
<td>PS-96
<td>Mining Functional Domain Sharing in Protein Structures
<td>Horng, Jorng-Tzong and Huang, Hsien Da and Wang, Shih-Hsien and Tsou, Tsung-Shan and Liu, Baw-Jhiune and Kao, Cheng-Yan and Hwang, Jenn-Kang
</tr>

<tr>
<td>PS-97
<td>Evolution of Protein Tyrosine Phosphatase Superfamilies: Evidences from Structures and Genome Data
<td>Huang, Jing-Fei and Lin, Jun
</tr>

<tr>
<td>PS-98
<td>Automated Determination of Protein Structures from NMR Data by Iterative Analysis of Self-Consistent NOE Contact Patterns
<td>Huang, Y. Janet and Tejero, Roberto and Montelione, Gaetano T.
</tr>

<tr>
<td>ME-53
<td>Instructional Movie: cDNA Microarrays for Gene Expression Analysis
<td>Huber, Wolfgang and S&uuml;ltmann, Holger and Poustka, Annemarie and Sawitzki, G&uuml;nther
</tr>

<tr>
<td>ON-121
<td>A Mathematical Modeling of GO Hierarchy for Gene Expression Profiling Analysis
<td>Hur, Jung and Lee, Sung and Oh, Ji and Chung, Tae and Kim, Yang
</tr>

<tr>
<td>SA-17
<td>Construction of Integrated Database for Protein Kinases and Proteases
<td>Hur, Jung and Park, Dong and Kim, Yang
</tr>

<tr>
<td>ME-54
<td>Inferring Genetic Network from Microarray Gene Expression Data by Bayesian Network and Nonparametric Regression
<td>Imoto, Seiya and Goto, Takao and Miyano, Satoru
</tr>

<tr>
<td>RR-80
<td>A Heuristic Approach to Identify Common RNA Structural Motifs in Homologous Sequences
<td>Ji, Yongmei and Stormo, Gary D.
</tr>

<tr>
<td>PA-116
<td>Dynamics of Myb Gene Family Evolution in <i>Arabidopsis</i>
<td>Jia, Li and Jiang, Tao and Clegg, Michael
</tr>

<tr>
<td>GF-34
<td>Identification and Analysis of Alternative Splicing Events in the Human Genome
<td>Kan, Zhengyan and Gish, Warren and States, David
</tr>

<tr>
<td>SF-40
<td>Discovering Motifs in Amino Acid Sequences using a Modified PrefixSpan Method
<td>Kanbara, Tomoki and Mori, Yasuma and Kitakami, Hajime and Kuroki, Susumu and Yamazaki, Yukiko
</tr>

<tr>
<td>ME-55
<td>Genes in Cocaine Addiction
<td>Keskin, Ozlem and Munshi, Rajan and Bahar, Ivet
</tr>

<tr>
<td>SA-18
<td>ConPath: Views Scaffolds and Calculates the Relative Orders, Orientations and Gaps of Contigs by Using Fosmid
<td>Kim, Pan-Gyu and Jung, Haeyoung and Park, Kiejung
</tr>

<tr>
<td>SA-19
<td>Finding All Normalized Local Alignments Above a Threshold
<td>Kim, Sangtae and Sim, Jeong S. and Park, Heejin and Park, Kunsoo and Park, Hyunseok and Seo, Jeong-Sun
</tr>

<tr>
<td>ME-56
<td>Effects of Sparse Time-Series Data on the Prediction of Gene Networks: A Study with a Boolean Network System Simulator.
<td>Kim, Sohyoung and Weinstein, John N. and Grefenstette, John J.
</tr>

<tr>
<td>RS-36
<td>Ribosome Builder: A Software Project to Simulate the Ribosome in Real-Time
<td>Knight, William and Hill, Walter E. and Lodmell, J. Stephen
</tr>

<tr>
<td>DS-37
<td>Distributions of Thermodynamic Values for DNA Oligomers: A Theoretical Survey of Tm and Hybridization Extent
<td>Koehler, Ryan T. and Peyret, Nicolas
</tr>

<tr>
<td>ME-57
<td>An Analysis of Matched mRNA Measurements from the High-Density HU6800 and cDNA Microarrays
<td>Kuo, Winston Patrick and Jenssen, Tor-Kristian and Kim, Ju Han and Kohane, Isaac S. and Ohno-Machado, Lucila
</tr>

<tr>
<td>PS-99
<td>A Method to Assess the Performance of Knowledge-Based Potentials with Respect to Amino Acid Types.
<td>Kuznetsov, Igor and Rackovsky, S.
</tr>

<tr>
<td>SA-20
<td>How Can We Quantitatively Describe Growth of Proteome Complexity Due to Evolution?
<td>Kuznetsov, Vladimir A.
</tr>

<tr>
<td>RR-81
<td>Deciphering <i>cis</i>-Acting Regulatory Elements in Plant and Drosophila Promoter Sequences
<td>Lescot, Magali and Rombauts, Stephane and D&egrave;hais, Patrice and Thijs, Gert and Martin, David and Thieffry, Denis and Jacq, Bernard and Moreau, Yves and Rouz&egrave;, Pierre and van Helden, Jacques
</tr>

<tr>
<td>RR-82
<td>Identification of <i>cis</i>-Regulatory Elements and Their Combination in Mammalian Promoter Sequences
<td>Leykin, Igor and Kao, Ming-Chih and Wong, Wing Hung
</tr>

<tr>
<td>SA-21
<td>Double-End Clone Sequence Assembly: Some Statistical Issues
<td>Li, Xiaoman and Waterman, Michael S.
</tr>

<tr>
<td>PW-89
<td>MetEx Viewer: A Visualization System for Metabolomics and Transcriptomics
<td>Li, Xing Jing and Lee, Christine and Hoops, Stefan and Mendes, Pedro
</tr>

<tr>
<td>PS-100
<td>Modeling Protein Flexibility for Structure-Based Active Site Redesign
<td>Lilien, Ryan H. and Anderson, Amy C. and Donald, Bruce R.
</tr>

<tr>
<td>SW-124
<td>The Design of an Agent-Based Workflow Tool for Bio-informatics
<td>Liu, Chung-Shyan and Ch'ang, Lan-Yang and Lin, Wen-Chang
</tr>

<tr>
<td>SA-22
<td>Remote Homology Detection Using Feature Vectors Formed Using Alignments of Small Motifs
<td>Logan, Beth and Moreno, Pedro and Suzek, Baris and Weng, Zhiping and Kasif, Simon
</tr>

<tr>
<td>ME-58
<td>Reconstructing the Temporal Ordering of Biological Samples Using Microarray Data
<td>Magwene, Paul M. and Lizardi, Paul and Kim, Junhyong
</tr>

<tr>
<td>SF-41
<td>Fast Algorithm for Extracting Multiple Unordered Short Motifs
<td>Maruyama, Osamu and Bannai, Hideo and Tamada, Yoshinori and Kuhara, Satoru and Miyano, Satoru
</tr>

<tr>
<td>PW-90
<td>Genomic Object Net: On-going Report on Biopathway Modeling and Simulation
<td>Matsuno, Hiroshi and Hirata, Yuichi and Doi, Atsushi and Miyano, Satoru
</tr>

<tr>
<td>ME-59
<td>Gene Expression Analysis by a Novel Dynamically Extendable Self-Organized Map Integrating Unsupervised and Supervised Learning
<td>Mavroudi, Seferina and Papadimitriou, Stergios and Bezerianos, Anastasios
</tr>

<tr>
<td>PS-101
<td>Extraction of Functionally Relevant Structural Templates from Protein Structures
<td>Milik, Mariusz and Szalma, Sandor and Olszewski, Krzystof A.
</tr>

<tr>
<td>SA-23
<td>Development of the Bacteria Analysis System on G-language Genome Analysis Environment
<td>Mori, Koya and Arakawa, Kazuharu and Tomita, Masaru
</tr>

<tr>
<td>ON-122
<td>Building Front End to the Common-Sense Genome Ontology
<td>Moucka, Gina and Galitsky, Boris
</tr>

<tr>
<td>PA-117
<td>Comparative Structural Phylogeny of Potassium Channels
<td>Palaniappan, Ashok and Jakobsson, Eric G.
</tr>

<tr>
<td>PS-102
<td>Structural Constrained Protein Evolution Simulations of Members of the Left-Handed Beta Helix Superfamily
<td>Parisi, Gustavo and Echave, Juli&aacute;n
</tr>

<tr>
<td>ME-60
<td>Principal Variables Analysis of Microarray Data
<td>Park, Peter J. and Kohane, Isaac S.
</tr>

<tr>
<td>PW-91
<td>A Fast and Robust Method to Infer and Characterize an Active Regulator Set for Molecular Pathways
<td>Pe'er, Dana and Regev, Aviv and Tanay, Amos
</tr>

<tr>
<td>RR-83
<td>Regulation of the Heat Shock Response of $\beta$-, $\gamma$- and $\epsilon$- Proteobacteria
<td>Permina, Elizabeth and Gelfand, Mikhail
</tr>

<tr>
<td>GF-35
<td>Stable RNA Secondary Structure of Human Donor Splice Sites
<td>Pervouchine, Dmitri and Graber, Joel and Kasif, Simon
</tr>

<tr>
<td>SA-24
<td>STROMA: Structurally Optimized Score Matrix for Sequence Alignment
<td>Qian, Bin and Goldstein, Richard A.
</tr>

<tr>
<td>PA-118
<td>A New Formulation of Evolutionary Models
<td>Rahmann, Sven and M&uuml;ller, Tobias
</tr>

<tr>
<td>SF-42
<td>Using Gibbs Sampling Strategies to Identify Motif Families in Networks of Interacting Proteins
<td>Reiss, David J. and Schwikowski, Benno
</tr>

<tr>
<td>ME-61
<td>Gene Expression Based Class Discovery of Familial Non-BRCA1/BRCA2 Breast Cancer
<td>Ringn&eacute;r, Markus and Hedenfalk, Ingrid and Ben-Dor, Amir and Yakhini, Zohar and Chen, Yidong and Meltzer, Paul and Borg, &Aring;ke and Trent, Jeffrey
</tr>

<tr>
<td>PS-103
<td>Computing Mutational Constraints Within Protein Domains Using Naturally-Occurring Sequence Variation
<td>Ruby, J. Graham and Zadeh, Joseph N. and Kao, Ming-Yang
</tr>

<tr>
<td>RR-84
<td>Analysis of Topological Representations of Transcriptional Regulatory Regions
<td>Sand, Olivier and Vu, Tien Deng and Gilbert, David and Viksna, Juris
</tr>

<tr>
<td>SA-25
<td>Development of Gene Ontology Analysis and Classification Tools for Microbial Genome Annotation
<td>Seo, Hwajung and Kim, Ki-Bong and Tae, Hongseok and Park, Wan and Park, Kiejung
</tr>

<tr>
<td>PS-104
<td>Compression of Protein Conformational Space
<td>Shao, Yu and Magdon-Ismail, Malik and Freedman, Daniel and Akella, Srinivas and Zaki, Mohammed and Bystroff, Chris
</tr>

<tr>
<td>PS-105
<td>Optimal Knowledge-Based Potentials and Threading Score Functions Derived from the Most Informative Sequence-Dependent Structural Probability Distributions
<td>Solis, Armando D. and Rackovsky, S.
</tr>

<tr>
<td>PS-106
<td>Using Solvation Model with Implicit Membrane for Transmembrane Protein Structure Modeling
<td>Spassov, Velin and Yan, Lisa and Szalma, Sandor
</tr>

<tr>
<td>SA-26
<td>PIR Non-Redundant Reference Protein Database (PIR-NREF)
<td>Suzek, Baris E. and Huang, Hongzhan and Orcutt, Bruce C. and Chen, Yongxing and Hu, Zhang-Zhi and Zhang, Jian and Wu, Cathy H.
</tr>

<tr>
<td>SA-27
<td>Two Visualization Tools for Sequence Contig Assembly
<td>Tae, Hongseok and Kim, Pan-Gyu and Park, Wan and Park, Kiejung
</tr>

<tr>
<td>ME-62
<td>XDotsReader, a Software for Automatic Analysis of cDNA Array Images
<td>Tahi, Fariza and Achddou, Brice and Decraene, Charles and Auffray, Charles and Pi&eacute;tu, Genevi&egrave;ve
</tr>

<tr>
<td>PS-107
<td>Hierarchical Machine Learning for Characterising FAD-binding Proteins
<td>Tan, Aik Choon and Gilbert, David
</tr>

<tr>
<td>DS-38
<td>Primer Tailing in Multiplex PCR
<td>Tang, Mengxiang
</tr>

<tr>
<td>PA-119
<td>GRIMM: Genome Rearrangements Software
<td>Tesler, Glenn
</tr>

<tr>
<td>PW-92
<td>Homology Searches for Pathways
<td>Tillinghast, John and Yang, Grace and Piercy, Chris
</tr>

<tr>
<td>ME-63
<td>Nonparametric Methods for Identifying Differentially Expressed Genes in Microarray Data
<td>Troyanskaya, Olga G. and Garber, Mitchell E. and Altman, Russ B. and Botstein, David
</tr>

<tr>
<td>ME-64
<td>An Intuitive Way for Selecting Informative Genes From Microarray Data
<td>Tsai, Huai-Kuang and Chao, Yu-Chih and Chuang, Han-Yu and Kao, Cheng-Yan
</tr>

<tr>
<td>SN-43
<td>Methods for Analysis and Visualization of SNP Genotype Data for Complex Diseases
<td>Tsalenko, Anya and Ben-Dor, Amir and Cox, Nancy and Yakhini, Zohar
</tr>

<tr>
<td>SA-28
<td>Aligning Two Fragmented Sequences
<td>Veeramachaneni, Vamsi and Berman, Piotr and Miller, Webb
</tr>

<tr>
<td>PS-108
<td>Sequence and Structural Analysis of APE1 Protein Family Using Physical-chemial Properties Based Motifs
<td>Venkatarajan, Mathura S. and Schein, Catherine H. and &Ouml;zg&uuml;n, Numan and Braun, Werner
</tr>

<tr>
<td>GF-33
<td>A Comparative Genomic Method for Computational Identification of Prokaryotic Translation Initiation Sites
<td>Walker, Megon and Pavlovic, Valdimir and Kasif, Simon
</tr>

<tr>
<td>SA-29
<td>A Framework Towards Efficient and Effective Protein Clustering
<td>Wang, Wei and Yang, Jiong
</tr>

<tr>
<td>SA-30
<td>Accelerating Search of Approximate Match on Large Protein Sequence Databases
<td>Wang, Wei and Yang, Jiong and Xia, Yi and Yu, Philip
</tr>

<tr>
<td>SA-31
<td>Pfam-A HMMs and SVMs for Protein Kinase Classification
<td>Weber, Ryan J. and New, Michael H.
</tr>

<tr>
<td>SA-32
<td>Finding Approximate Tandem Repeats in Genomic Sequences
<td>Wexler, Ydo and Kashi, Yechezkel and Geiger, Dan
</tr>

<tr>
<td>ME-65
<td>How Useful are the Bayesian Networks and Dynamic Bayesian Networks for Modeling Gene Expression Data
<td>Wu, Baolin and Zhao, Hongyu
</tr>

<tr>
<td>PS-109
<td>Combinatorial Optimization Algorithms for Assignment of Protein Backbone NMR Peaks
<td>Xu, Ying and Xu, Dong and Kim, Dongsup and Chen, Zhi-Zhong and Lin, Guohui and Jiang, Tao
</tr>

<tr>
<td>ME-66
<td>A New Framework for Biological Data Clustering Using Minimum Spanning Trees
<td>Xu, Ying and Xu, Dong and Olman, Victor
</tr>

<tr>
<td>ME-67
<td>Improving Performance of Bicluster Discovery in a Large Data Set
<td>Yang, Jiong and Wang, Wei and Wang, Haixun and Yu, Philip
</tr>

<tr>
<td>ME-68
<td>Feature Selection and Clustering for Tumor Classification
<td>Yeang, Chen-Hsiang
</tr>

<tr>
<td>MA-46
<td>Relevance Network between Transcriptome and Chemosensitivity in Hepatoma Cells
<td>Yujin, Hoshida and Masaru, Moriyama and Motoyuki, Otsuka and Nishimura, Shin-ichiro and Naoya, Kato and Yasushi, Shiratori and Naohiko, Seki and Masao, Omata
</tr>

<tr>
<td>PS-110
<td>Solving Molecular Inverse Kinematics Problems for Protein Folding and Drug Design
<td>Zhang, Ming and Kavraki, Lydia E.
</tr>

<tr>
<td>ME-69
<td>A Unified Maximum Separability Analysis Approach for Biological Expression Data Processing
<td>Zhang, Zhen and Zhang, Hong and Chan, Daniel W.
</tr>

<tr>
<td>PS-111
<td>Protein Structure Prediction Aided by Solution Xray Scattering Data, and Its Application in Structural Homology Study
<td>Zheng, Wenjun and Doniach, Sebastian
</tr>

<tr>
<td>PW-93
<td>Computational Identification of Operons in Microbial Genomes
<td>Zheng, Yu and Roberts, Richard J. and Kasif, Simon
</tr>

<tr>
<td>ME-70
<td>Construct Regulatory Networks from Gene Expression Data Using Association Rule Mining
<td>Zhou, Jiaye and Veroff, Robert L. and Helman, Paul
</tr>

<tr>
<td>ME-71
<td>Prediction of Gene Function by Expression Transitivity using Microarray Data
<td>Zhou, Xianghong and Kao, Ming-Chih Jeffrey and Wong, Wing Hung
</tr>

<tr>
<td>ME-72
<td>Finding Regulatory Elements from Microarray data
<td>Zilberstein, Chaya Ben-Zaken and Geiger, Dan
</tr>

<tr>
<td>ON-120
<td>FunCatTM Controlled Vocabulary for the Human Genome: Enhancing the MIPS Yeast Functional Catalogue to Encompass the Biology of Prokaryotes, Plants and Animals
<td>Zollner, Alfred and Albermann, Kaj and Hani, Jean and Ehrenreich, Armin and Reinfried, Goetz and Maier, Dieter and Mayer, Klaus and Schoof, Heiko and Mewes, Hans-Werner
</tr>

<tr>
<td>ME_73
<td>Exploration of Yeast Gene Expression Regulation via Classification.
<td>van Zwet, Erik and Kechris, Katherina and Chiang, Derek and Bickel, Peter and Eisen, Michael
</tr>

</table>
</body>
</html>

