<BASE HREF="http://www.erpland.org/recomb/">
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">

<html>
<head>
<title>RECOMB 2002 Schedule</title>
<LINK rel="stylesheet" href="styles/styleNN4.css">
</HEAD>

<BODY bgcolor="#ffffff" bottommargin="0" leftmargin="0" marginheight="0" marginwidth="0" topmargin="0" rightmargin="0">

<TABLE border="0" cellpadding="0" cellspacing="0" width="762">
<TR valign="top">
<TD width="10"><IMG src="images/spacer.gif" height=1 width=10></TD>
<TD width="562">
		
<center>		
<H1 class="a17bBlack">The RECOMB 2002 Schedule</H1>
</center>

<!--
<P>
You can also download printable <A HREF="RECOMB2002Schedule.ps">PostScript</A>
or <A HREF="RECOMB2002Schedule.pdf">PDF</A> versions.
</P>
!-->

<HR><H1>Wednesday, April 17.</H1>
<TABLE>
<TR><TD VALIGN=TOP>18:00--20:00</TD><TD VALIGN=TOP><B>Registration and Reception</B></TD></TR>
</TABLE>

<HR><H1>Thursday, April 18.</H1>
<TABLE>
<TR><TD VALIGN=TOP>8:45--9:00</TD><TD VALIGN=TOP><B>Opening:</B>
Gene Myers, Program Chair; Sridhar Hannenhalli, Conference Chair; and Martin Vingron, RECOMB 2003 Conference Chair</TD></TR>
<TR><TD VALIGN=TOP>9:00--9:50</TD><TD VALIGN=TOP><B>The Stanislaw Ulam Memorial Lecture</B>: Gene Myers, Chair<BR>Sequencing the Human Genome<BR><I>J. Craig Venter</I></TD></TR>
<TR><TD VALIGN=TOP>9:50--10:15</TD><TD VALIGN=TOP>Coffee Break</TD></TR>
<TR><TD VALIGN=TOP></TD><TD VALIGN=TOP><FONT SIZE=+1><B>MOTIFS:</B></FONT>
Isidore Rigoutsos, Chair</TD></TR>
<TR><TD VALIGN=TOP>10:15--10:40</TD><TD VALIGN=TOP>Monotony of Surprise And Large-Scale Quest for Unusual Words <BR><I>Alberto Apostolico</I>, <I>Mary Ellen Bock</I>, <I>Stefano Lonardi</I></TD></TR>
<TR><TD VALIGN=TOP>10:40--11:05</TD><TD VALIGN=TOP>Finding Motifs in the Twilight Zone<BR><I>Uri Keich</I>, <I>Pavel A. Pevzner</I></TD></TR>
<TR><TD VALIGN=TOP>11:05--11:30</TD><TD VALIGN=TOP>Discriminative Motifs<BR><I>Saurabh Sinha</I></TD></TR>
<TR><TD VALIGN=TOP>11:30--11:55</TD><TD VALIGN=TOP>Trilogy: Discovery of Sequence-Structure Patterns Across Diverse Proteins<BR><I>Phil Bradley</I>, <I>Peter S. Kim</I>, <I>Bonnie Berger</I></TD></TR>
<TR><TD VALIGN=TOP>11:55--13:30</TD><TD VALIGN=TOP>Lunch Break</TD></TR>
<TR><TD VALIGN=TOP>13:30--14:20</TD><TD VALIGN=TOP><B>Keynote</B>: Sorin Istrail, Chair<BR>Toward a Proteome Atlas for <I>C. elegans</I><BR><I>Marc Vidal</I></TD></TR>
<TR><TD VALIGN=TOP>14:20--14:45</TD><TD VALIGN=TOP>Coffee Break</TD></TR>
<TR><TD VALIGN=TOP></TD><TD VALIGN=TOP><FONT SIZE=+1><B>GENE EXPRESSION:</B></FONT>
Terry Gaasterland, Chair</TD></TR>
<TR><TD VALIGN=TOP>14:45--15:10</TD><TD VALIGN=TOP>Phase-Independent Rhythmic Analysis of Genome-Wide Expression Patterns<BR><I>Christopher James Langmead</I>, <I>Anthony K. Yan</I>, <I>C. Robertson McClung</I>, <I>Bruce Randall Donald</I></TD></TR>
<TR><TD VALIGN=TOP>15:10--15:35</TD><TD VALIGN=TOP>A New Approach to Analyzing Gene Expression Time Series Data<BR><I>Ziv Bar-Joseph</I>, <I>Georg Gerber</I>, <I>David K. Gifford</I>, <I>Tommi S. Jaakkola</I>, <I>Itamar Simon</I></TD></TR>
<TR><TD VALIGN=TOP>15:35--16:00</TD><TD VALIGN=TOP>A Bayesian Approach to Transcript Estimation from Gene Array Data: The BEAM Technique<BR><I>Ron O. Dror</I>, <I>Jonathan G. Murnick</I>, <I>Nicola A. Rinaldi</I></TD></TR>
<TR><TD VALIGN=TOP>16:00--16:25</TD><TD VALIGN=TOP>Coffee Break</TD></TR>
<TR><TD VALIGN=TOP>16:25--16:50</TD><TD VALIGN=TOP>Microarrays: How Many Do You Need?<BR><I>Alexander Zien</I>, <I>Juliane Fluck</I>, <I>Ralf Zimmer</I>, <I>Thomas Lengauer</I></TD></TR>
<TR><TD VALIGN=TOP>16:50--17:15</TD><TD VALIGN=TOP>Analysis of Gene Expression Profiles: Class Discovery and Leaf Ordering <BR><I>Chris H.Q. Ding</I></TD></TR>
<TR><TD VALIGN=TOP>17:15--17:40</TD><TD VALIGN=TOP>Discovering Local Structure in Gene Expression Data: The Order-Preserving Submatrix Problem<BR><I>Amir Ben-Dor</I>, <I>Benny Chor</I>, <I>Richard Karp</I>, <I>Zohar Yakhini</I></TD></TR>
<TR><TD VALIGN=TOP>21:00--22:30</TD><TD VALIGN=TOP><B>RECOMB Business Meeting and Awards</B> (open for all attendees)</TD></TR>
</TABLE>

<HR><H1>Friday, April 19.</H1>
<TABLE>
<TR><TD VALIGN=TOP>9:00--9:50</TD><TD VALIGN=TOP><B>The Distinguished New Technologies Lecture</B>: Thomas Lengauer, Chair<BR>Applied Computational Genomics:  Polymorphism Prediction, Data Mining and Genomic Analysis<BR><I>Harold ``Skip'' Garner</I></TD></TR>
<TR><TD VALIGN=TOP>9:50--10:15</TD><TD VALIGN=TOP>Coffee Break</TD></TR>
<TR><TD VALIGN=TOP></TD><TD VALIGN=TOP><FONT SIZE=+1><B>FUNCTION:</B></FONT> George Komatsoulis, Chair</TD></TR>
<TR><TD VALIGN=TOP>10:15--10:40</TD><TD VALIGN=TOP>Combining Pairwise Sequence Similarity and Support Vector Machines for Remote Protein Homology Detection<BR><I>Li Liao</I>, <I>William Stafford Noble</I></TD></TR>
<TR><TD VALIGN=TOP>10:40--11:05</TD><TD VALIGN=TOP>Probabilistic Hierarchical Clustering for Biological Data<BR><I>Eran Segal</I>, <I>Daphne Koller</I></TD></TR>
<TR><TD VALIGN=TOP>11:05--11:30</TD><TD VALIGN=TOP>Inferring Domain-Domain Interactions from Protein-Protein Interactions<BR><I>Minghua Deng</I>, <I>Shipra Mehta</I>, <I>Fengzhu Sun</I>, <I>Ting Chen</I></TD></TR>
<TR><TD VALIGN=TOP>11:30--11:55</TD><TD VALIGN=TOP>From Promoter Sequence to Expression: A Probabilistic Framework<BR><I>Eran Segal</I>, <I>Yoseph Barash</I>, <I>Itamar Simon</I>, <I>Nir Friedman</I>, <I>Daphne Koller</I></TD></TR>
<TR><TD VALIGN=TOP>11:55--13:30</TD><TD VALIGN=TOP>Lunch Break</TD></TR>
<TR><TD VALIGN=TOP>13:30--14:20</TD><TD VALIGN=TOP><B>Keynote</B>: Ron Shamir, Chair<BR>Computational Structural Proteomics and Virtual Ligand Screening<BR><I>Ruben Abagyan</I></TD></TR>
<TR><TD VALIGN=TOP>14:20--14:45</TD><TD VALIGN=TOP>Coffee Break</TD></TR>
<TR><TD VALIGN=TOP></TD><TD VALIGN=TOP><FONT SIZE=+1><B>PROTEIN STRUCTURE AND MOTION:</B></FONT> Hans-Peter Lenhoff, Chair</TD></TR>
<TR><TD VALIGN=TOP>14:45--15:10</TD><TD VALIGN=TOP>Structural Alignment of Large-Size Proteins via Lagrangian Relaxation <BR><I>Alberto Caprara</I>, <I>Giuseppe Lancia</I></TD></TR>
<TR><TD VALIGN=TOP>15:10--15:35</TD><TD VALIGN=TOP>Revealing Protein Structures: A New Method for Mapping Antibody Epitopes<BR><I>Brendan M. Mumey</I>, <I>Brian W. Bailey</I>, <I>Edward A. Dratz</I></TD></TR>
<TR><TD VALIGN=TOP>15:35--16:00</TD><TD VALIGN=TOP>Stochastic Roadmap Simulation: An Efficient Representation and Algorithm for Analyzing Molecular Motion<BR><I>Mehmet Serkan Apaydin</I>, <I>Douglas L. Brutlag</I>, <I>Carlos Guestrin</I>, <I>David Hsu</I>, <I>Jean-Claude Latombe</I></TD></TR>
<TR><TD VALIGN=TOP>16:00--16:25</TD><TD VALIGN=TOP>A Dimensionality Reduction Approach to Modeling Protein Flexibility<BR><I>Miguel L. Teodoro</I>, <I>George N. Phillips Jr.</I>, <I>Lydia E. Kavraki</I></TD></TR>
<TR><TD VALIGN=TOP>16:25--16:50</TD><TD VALIGN=TOP>Using Motion Planning to Map Protein Folding Landscapes and Analyze Folding Kinetics of Known Native Structures<BR><I>Nancy M. Amato</I>, <I>Ken A. Dill</I>, <I>Guang Song</I></TD></TR>
<TR><TD VALIGN=TOP>16:50--19:00</TD><TD VALIGN=TOP><B>Poster Session and Software Demonstrations</B></TD></TR>
</TABLE>

<HR><H1>Saturday, April 20.</H1>
<TABLE>
<TR><TD VALIGN=TOP>9:00--9:50</TD><TD VALIGN=TOP><B>The Distinguished Biology Lecture</B>: Michael Waterman, Chair<BR>Lymphocyte Turnover in HIV-1 Infection and the Role of the Thymus in SIV Infection<BR><I>David D. Ho</I></TD></TR>
<TR><TD VALIGN=TOP>9:50--10:15</TD><TD VALIGN=TOP>Coffee Break</TD></TR>
<TR><TD VALIGN=TOP></TD><TD VALIGN=TOP><FONT SIZE=+1><B>SEQUENCE ANALYSIS:</B></FONT> Serafim Batzoglou, Chair</TD></TR>
<TR><TD VALIGN=TOP>10:15--10:40</TD><TD VALIGN=TOP>Provably Sensitive Indexing Strategies for Biosequence Similarity Search<BR><I>Jeremy Buhler</I></TD></TR>
<TR><TD VALIGN=TOP>10:40--11:05</TD><TD VALIGN=TOP>Picking Alignments from (Steiner) Trees<BR><I>Lior Pachter</I>, <I>Fumei Lam</I>, <I>Marina Alexandersson</I></TD></TR>
<TR><TD VALIGN=TOP>11:05--11:30</TD><TD VALIGN=TOP>Comparison of Minisatellites<BR><I>S\`everine B\'erard</I>, <I>\'Eric Rivals</I></TD></TR>
<TR><TD VALIGN=TOP>11:30--11:55</TD><TD VALIGN=TOP>The Restriction Scaffold Problem<BR><I>Amir Ben-Dor</I>, <I>Richard M. Karp</I>, <I>Benno Schwikowski</I>, <I>Ron Shamir</I></TD></TR>
<TR><TD VALIGN=TOP>11:55--13:30</TD><TD VALIGN=TOP>Lunch Break</TD></TR>
<TR><TD VALIGN=TOP>13:30--14:20</TD><TD VALIGN=TOP><B>Keynote</B>: Pavel Pevzner, Chair<BR>Microarray Analysis of Vertebrate Embryonic Neural Induction<BR><I>Ali H. Brivanlou</I></TD></TR>
<TR><TD VALIGN=TOP>14:20--14:45</TD><TD VALIGN=TOP>Coffee Break</TD></TR>
<TR><TD VALIGN=TOP></TD><TD VALIGN=TOP><FONT SIZE=+1><B>SNPs AND EVOLUTION:</B></FONT>
Benny Chor, Chair</TD></TR>
<TR><TD VALIGN=TOP>14:45--15:10</TD><TD VALIGN=TOP>DNA Sequence Evolution with Neighbor-Dependent Mutation<BR><I>Peter F. Arndt</I>, <I>Christopher B. Burge</I>, <I>Terence Hwa</I></TD></TR>
<TR><TD VALIGN=TOP>15:10--15:35</TD><TD VALIGN=TOP>Significance of Inter-Species Matches When Evolutionary Rate Varies<BR><I>Jia Li</I>, <I>Webb Miller</I></TD></TR>
<TR><TD VALIGN=TOP>15:35--16:00</TD><TD VALIGN=TOP>Deconvolving Sequence Variation in Mixed DNA Populations<BR><I>Andy Wildenberg</I>, <I>Steven Skiena</I>, <I>Pavel Sumazin</I></TD></TR>
<TR><TD VALIGN=TOP>16:00--16:25</TD><TD VALIGN=TOP>Haplotyping as Perfect Phylogeny: Conceptual Framework and Efficient Solutions <BR><I>Dan Gusfield</I></TD></TR>
<TR><TD VALIGN=TOP>16:25--16:50</TD><TD VALIGN=TOP>Set Association Analysis of SNP Case-Control and Microarray Data<BR><I>Jurg Ott</I>, <I>Josephine Hoh</I></TD></TR>
<TR><TD VALIGN=TOP>16:50--19:00</TD><TD VALIGN=TOP><B>Poster Session and Software Demonstrations</B></TD></TR>
</TABLE>

<HR><H1>Sunday, April 21.</H1>
<TABLE>
<TR><TD VALIGN=TOP>9:00--9:50</TD><TD VALIGN=TOP><B>Keynote</B>: Sridhar Hannenhalli, Chair<BR>Recent Duplication, Evolution and Assembly of the Human Genome<BR><I>Evan E. Eichler</I></TD></TR>
<TR><TD VALIGN=TOP>9:50--10:15</TD><TD VALIGN=TOP>Coffee Break</TD></TR>
<TR><TD VALIGN=TOP></TD><TD VALIGN=TOP><FONT SIZE=+1><B>RNA and GENOME REARRANGEMENTS:</B></FONT> 
Marie-France Sagot, Chair</TD></TR>
<TR><TD VALIGN=TOP>10:15--10:40</TD><TD VALIGN=TOP>Approximate Matching of Secondary Structures <BR><I>Nadia El-Mabrouk</I>, <I>Mathieu Raffinot</I></TD></TR>
<TR><TD VALIGN=TOP>10:40--11:05</TD><TD VALIGN=TOP>Designing RNA Structures: Natural and Artificial Selection <BR><I>Barry Cohen</I>, <I>Steven Skiena</I></TD></TR>
<TR><TD VALIGN=TOP>11:05--11:30</TD><TD VALIGN=TOP>An Algorithm to Find All Sorting Reversals<BR><I>Adam C. Siepel</I></TD></TR>
<TR><TD VALIGN=TOP>11:30--11:55</TD><TD VALIGN=TOP>Tests for Gene Clustering<BR><I>Dannie Durand</I>, <I>David Sankoff</I></TD></TR>
<TR><TD VALIGN=TOP>11:55--13:30</TD><TD VALIGN=TOP>Lunch Break</TD></TR>
<TR><TD VALIGN=TOP>13:30--14:20</TD><TD VALIGN=TOP><B>Keynote</B> 
David Sankoff, Chair<BR>Biological and Computational Annotation of the Drosophila Genome Sequence<BR><I>Gerald M. Rubin</I></TD></TR>
<TR><TD VALIGN=TOP>14:20--14:45</TD><TD VALIGN=TOP>Coffee Break</TD></TR>
<TR><TD VALIGN=TOP></TD><TD VALIGN=TOP><FONT SIZE=+1><B>DNA HYBRIDIZATION:</B></FONT>
Vineet Bafna, Chair</TD></TR>
<TR><TD VALIGN=TOP>14:45--15:10</TD><TD VALIGN=TOP>Sequencing by Hybridization using Direct and Reverse Cooperating Spectra<BR><I>Samuel A. Heath</I>, <I>Franco P. Preparata</I>, <I>Joel Young</I></TD></TR>
<TR><TD VALIGN=TOP>15:10--15:35</TD><TD VALIGN=TOP>Handling Long Targets and Errors in Sequencing by Hybridization<BR><I>Eran Halperin</I>, <I>Shay Halperin</I>, <I>Tzvika Hartman</I>, <I>Ron Shamir</I></TD></TR>
<TR><TD VALIGN=TOP>15:35--16:00</TD><TD VALIGN=TOP>String Barcoding: Uncovering Optimal Virus Signatures<BR><I>Sam Rash</I>, <I>Dan Gusfield</I></TD></TR>
<TR><TD VALIGN=TOP>16:00--16:15</TD><TD VALIGN=TOP><B>End of Conference</B></TD></TR>
</TABLE>

</TD>
</TR>		
</TABLE>
</BODY>
</HTML>

